stable/nginx-ingress/templates/controller-service.yaml

```diff
@@ -63,18 +63,22 @@ spec:
       port: {{ $key }}
       protocol: TCP
       targetPort: "{{ $key }}-tcp"
+     {{- if $.Values.controller.service.nodePorts.tcp }}
```

Comment:
I am not sure what purpose these added if statements serve. The next line should handle this.

Reply:
because when my value ```.Values.controller.service.nodePorts.tcp``` is ```null``` while my value ```.Values.tcp``` is not ```null``` I'm getting the error:
```
helm template . --set tcp[0]='27017: "default/mongodb:27017"'  --set tcp[1]='15671: "default/rabbitmq:15671"'

Error: render error in "nginx-ingress/templates/controller-service.yaml": template: nginx-ingress/templates/controller-service.yaml:66:13: executing "nginx-ingress/templates/controller-service.yaml" at <index $.Values.controller.service.nodePorts.tcp $key>: error calling index: value has type int; should be string
```